launcher/build.gradle.kts

```diff
@@ -48,7 +48,13 @@ dependencies {
 
     // Blob storage container provisioning
     implementation("${edcGroup}:blobstorage:${edcVersion}")
-    implementation("${edcGroup}:azure-vault:${edcVersion}")
+    // To use FileSystem vault e.g. -DuseFsVault="true".Only for non-production usages.
+    val useFsVault: Boolean = System.getProperty("useFsVault", "false").toBoolean()
```

Comment:
i think using the FS vault should be the default. eventually, we'll likely need multiple launcher configs instead of build switchtes, e.g. once we start using actual persistence resources (cosmos, postgres,...).

Reply:
yeah, not a big deal. once the different configs diverge more, eg. cosmos vs in-mem stores, blobstore vs local filesystem, etc. we will have to create more launcher configs anyway. not necessary for now.